style.css

```diff
@@ -131,5 +145,190 @@ h1 {
   justify-content: space-between;
   align-items: center;
   width: 50%;
-  margin-bottom: 40%;
+  margin-bottom: 30%;
+}
+
+.project-main-wrapper {
+  background-color: #fff;
+  border-radius: 20px;
+  margin: auto 27px;
+}
```

Comment:
Kindly make sure that your project sticks as much as possible to the Figma template provided. Kindly increase the top margin of your image card. You could get this value from the Figma template provided. :eyes: :eyes: 
Your design
![image](https://user-images.githubusercontent.com/98363075/198752123-45b198b2-1841-41d5-9b9b-14fced3d2b10.png)

Figma design
![image](https://user-images.githubusercontent.com/98363075/198752026-492b0c8e-874b-4e5f-8894-31108bfdaf5d.png)